site/_quarto.yml

```diff
@@ -36,10 +36,10 @@ website:
         file: get-started/get-started.qmd
       - text: "Guides"
         file: guide/guides.qmd
-      - text: "{{< fa cube >}} Developer Framework"
+      - text: "{{< fa cube >}} Developer"
```

Comment:
```suggestion
      - text: "{{< fa cube >}} Developers"
```

I wonder if we don't need something a bit more here? My first reaction was "Developer ... _what_?" The text feels incomplete. 

How about one of these?

- `Developers`  — probably the simplest tweak that resolves the issue and my suggestion
- `For Developers` — highlights the role more explicitly
- `Developer Documentation` — more in line with the other navbar items


![image](https://github.com/user-attachments/assets/5fc4e57b-5314-4749-8b65-3e091c519a26)

Reply:
https://github.com/validmind/documentation/blob/2c7be4f11513edaa5eb1d54bd96208b4f3e86f13/site/_quarto.yml#L39